src/_extensions/acceptance-criteria.md

```diff
@@ -47,6 +66,10 @@ Your extension must satisfy all the following acceptance criteria to be included
 - Values in `manifest.json` must be sound and valid.
 - Flash is not allowed.
 - We must be able to review the code in a reasonable manner. Therefore, the code shouldn’t be obfuscated. Binary code is not ok.
+- There must be no unused files.
+- There must be no redundant permission requested or manifest entries.
+- Third-party libraries must be in genuine and in unchanged form. They should be acquired from vendor’s official website or a CDN.
```

Comment:
Missing article. "the vendor's" or "a vendor's". 

Maybe they should also be required to be "well known" or I can just create a "library" myself.